book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_reader :rental
+  attr_accessor :title, :author
```

Comment:
- [ OPTIONAL ] Hey there! 👋 It's widely embraced as a best community practice to place your getters and setters at the top of your class definition. 🚀 This helps in providing a clear and organized structure to your class, making it easier for developers to quickly understand and access the properties of the class. 📚💻 Keep your code clean and readable! ✨